index.rst

```diff
@@ -85,5 +85,7 @@ visit our `gallery </docs/dev/auto_examples>`__.
          :class: coins-sample span6
 
 You can read more in our `user guide </docs/stable/user_guide.html>`__.
+For a proper introduction to image processing, you may want to check out this
```

Comment:
Perhaps:

For an introduction to image processing using scikit-image, see
`this lesson <...>` by Data Carpentry.

Reply:
Yes, your wording is more neutral, thanks! Applied: f57948eb8eb62